portfolio-project-matching-app/models/FilteredProject.js

```diff
@@ -1,10 +1,77 @@
 import { Project } from './Project'
 
 class FilteredProject {
+    // fields
+    project;
+    buttonFilters;
+    searchFiltered;
```

Comment:
Went ahead and created a new field/property called `searchFiltered`. Basically just a boolean indication as to whether the search has been applied to a given `FilteredProject` object.